app/src/main/res/layout/activity_about.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.activities.AboutActivity">
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_marginTop="@dimen/fab_margin"
+                android:layout_height="150dp"
+                android:layout_gravity="center"
+                android:src="@drawable/mentorship_system_logo"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="@color/black"
```

Comment:
Neither of the TextViews on the app, use color black explicitly, so you don't need to put it here, this may look weird, taking other textViews as an example. Do this for any other text view you have here.